odxtools/database.py

```diff
@@ -4,6 +4,7 @@
 from typing import List, Optional
 from xml.etree import ElementTree
 from zipfile import ZipFile
+
 from packaging.version import Version
```

Comment:
meaningless change

Reply:
Caused by reformat-source script

Reply:
@andlaus most likely caused by an upgrade (new version) of yapf